.github/workflows/release.yml

```diff
@@ -9,7 +9,6 @@ on:
 permissions:
```

Comment:
Worth moving these perms to the release please step so they dont apply to the rest of the workflow?